mllib-local/src/main/scala/org/apache/spark/ml/linalg/Matrices.scala

```diff
@@ -759,6 +769,8 @@ object SparseMatrix {
 
 /**
  * Factory methods for [[org.apache.spark.ml.linalg.Matrix]].
+ *
+ * @since 2.0
```

Comment:
Oh right, because that's actually an annotation in Spark Core and it's not available. My mistake. For now fair enough then. But do write 2.0.0